packages/ui/src/components/Navigation/components/LanguagesDropdown.tsx

```diff
@@ -0,0 +1,81 @@
+import { CheckIcon, GlobeAltIcon } from '@heroicons/react/24/outline';
+import type { Category, Culture, ExtendedStory, NewsroomLanguageSettings } from '@prezly/sdk';
+import { getLanguageDisplayName, getUsedLanguages, LocaleObject } from '@prezly/theme-kit-core';
+import { useMemo } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { Dropdown } from '@/components/Dropdown';
+
+import { getTranslationUrl } from '../util';
+
+export interface Props {
+    languages: NewsroomLanguageSettings[];
+    locale: Culture['code'];
+    currentStory?: ExtendedStory;
+    currentCategory?: Pick<Category, 'i18n' | 'display_name'>;
+    hasError?: boolean;
+}
+
+export function LanguagesDropdown({
```

Comment:
Languages dropdown is almost inaccessible on mobile screens:

<img width="457" alt="image" src="https://github.com/prezly/theme-kit-js/assets/1641218/7785b468-8701-473e-a654-d85fa43f0df8">